emulators/spim/Portfile

```diff
@@ -42,9 +42,11 @@ if {[string match *clang* ${configure.cxx}]} {
 }
 build.post_args \
     CC="${configure.cxx} ${cxx_stdlibflags}" \
-    EXCEPTION_DIR=${prefix}/share/spim
+    EXCEPTION_DIR=${prefix}/share/spim \
+    DESTDIR="${prefix}"
 
 destroot.post_args \
+    DESTDIR=${prefix} \
```

Comment:
This sounds suspicious to me. Ideally one should separately define `prefix` (`/opt/local`) and `destdir` (`/some/super/strange/temporary/path`);

And then you would supply `DESTDIR=${destroot}`.